blockchain/filedao/filedao_v2.go

```diff
@@ -48,11 +48,13 @@ type (
 		hashStore db.CountingIndex // store block hash
 		blkStore  db.CountingIndex // store raw blocks
 		sysStore  db.CountingIndex // store transaction log
+		cfg       Config
```

Comment:
we don't need to add `cfg` into the struct? the only new thing needed is `deser *block.Deserializer` below